.github/dependabot.yml

```diff
@@ -5,6 +5,7 @@ updates:
     schedule:
       interval: weekly
       day: tuesday
+    open-pull-requests-limit: 15
```

Comment:
15 is magical number for us to try - it doesn't overload PR list as default list limit is 20 and we can try how it works.

Reply:
let's try 10, we are hitting the limit because of the k8s stuff which is going unreviewed so we need to work that with the kube team

Reply:
Agree!

Reply:
I also saw Kubernetes being issue. I propose we do group some PRs in the future. 
Ths draft PR contains some ideas: https://github.com/mongodb/mongodb-atlas-cli/pull/2124